plugin/resolvergen/testdata/filetemplate/out/schema.custom.go

```diff
@@ -36,9 +36,11 @@ type resolverCustomResolverType struct{ *CustomResolverType }
 // !!! WARNING !!!
 // The code below was going to be deleted when updating resolvers. It has been copied here so you have
 // one last chance to move it out of harms way if you want. There are two reasons this happens:
-//   - When renaming or deleting a resolver the old code will be put in here. You can safely delete
-//     it when you're done.
-//   - You have helper methods in this file. Move them out to keep these resolver files clean.
-func AUserHelperFunction() {
+//  - When renaming or deleting a resolver the old code will be put in here. You can safely delete
+//    it when you're done.
+//  - You have helper methods in this file. Move them out to keep these resolver files clean.
+/*
+	func AUserHelperFunction() {
 	// AUserHelperFunction implementation
 }
+*/
```

Comment:
This change is causing one test to fail

Reply:
I reverted it but this change gets made automatically when I run `go test ./...`. I think we need to revert #3243.

Reply:
Ah, crud. huh. Either that, or rethink how we do testing for that behavior.

Reply:
Yeah potentially. Either way, I would revert for now, we can fix it later. I don't want `master` to be in a broken state. We should also update CI to ensure the git state isn't dirty after running the tests.